packages/frontend/src/components/AuthenticatedPage.tsx

```diff
@@ -1,11 +1,11 @@
 import React, { useEffect, useMemo } from 'react';
-import { getConfig } from 'backend/config/config';
 import { useRecoilState } from 'recoil';
 import { useNavigate } from 'react-router-dom';
+import { GetFrontendConfig } from '../config/config';
 import BackendUserClient, { AuthResponse } from '../api/users/client';
 import { UserState, UserIsAuthenticated } from '../state/store';
 
-const appConfig = getConfig();
+const frontendConfig = GetFrontendConfig();
```

Comment:
Why the capitalization change? Do you have a convention for capitalization of function names?

Reply:
Good callout. Im being inconsistent. Id maybe lean towards capitalized functions. Probably because its how Go works. But I am open to suggestions.

Do you follow a preferred convention for function naming?

Reply:
What I've generally done is `CapitalCase` for functions that are React components, and `lowerCase` for other functions. I think that's common in the React / JS world